src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -88,7 +87,7 @@ const styles = StyleSheet.create({
     fontStyle: 'normal',
     fontWeight: '500',
     fontSize: 14,
-    color: COLORS.black,
+    color: '#000000',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider using a centralized color management system instead of hardcoding color values.
```diff
- color: '#000000',
+ color: colors.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->